scripts/key-press.js

```diff
@@ -0,0 +1,6 @@
+export default function handleKeyPress(event, func, input) {
```

Comment:
this function currently only handles `Enter` and not any/all `keypress` events. it may be better to rename it.

also, this is very closely connected to pressing the Enter in an input field (since input is also an argument of the function). try to make it independent of the input, if you can :)